assets/keymaps/default-macos.json

```diff
@@ -613,8 +613,12 @@
       "enter": "project_panel::Rename",
       "backspace": ["project_panel::Trash", { "skip_prompt": false }],
       "delete": ["project_panel::Trash", { "skip_prompt": false }],
-      "cmd-backspace": ["project_panel::Delete", { "skip_prompt": false }],
-      "cmd-delete": ["project_panel::Delete", { "skip_prompt": false }],
+      "shift-backspace": ["project_panel::Delete", { "skip_prompt": false }],
+      "shift-delete": ["project_panel::Delete", { "skip_prompt": false }],
```

Comment:
cmd+backspace/delete and cmd+shift+backspace/delete look right.

Is there a reason we're adding shift+backspace and shift+delete? I've never seen them